framework/include/problems/FEProblemBase.h

```diff
@@ -1065,12 +1065,7 @@ class FEProblemBase : public SubProblem, public Restartable
   virtual void computeResidualTag(const NumericVector<Number> & soln,
                                   NumericVector<Number> & residual,
                                   TagID tag);
-  /**
-   * Form a residual vector for a given tag and "residual" tag
-   */
-  virtual void computeResidualType(const NumericVector<Number> & soln,
-                                   NumericVector<Number> & residual,
-                                   TagID tag);
+
```

Comment:
There was a reason to have this function at the first place, but I just forgot what it was

Reply:
`git grep` shows that it's not being used anywhere, at least not in the moose repository. I guess maybe apps might be calling it, which will show up in the testing